theano/sandbox/rng_mrg.py

```diff
@@ -1348,14 +1367,14 @@ def multinomial(self, size=None, n=1, pvals=None, ndim=None, dtype='int64',
 
         if size is not None:
             raise ValueError("Provided a size argument to "
-                             "MRG_RandomStreams.multinomial, which does not use "
-                             "the size argument.")
+                             "MRG_RandomStreams.multinomial, "
+                             "which does not use the size argument.")
         if ndim is not None:
             raise ValueError("Provided an ndim argument to "
-                             "MRG_RandomStreams.multinomial, which does not use "
-                             "the ndim argument.")
+                             "MRG_RandomStreams.multinomial, "
```

Comment:
This line seems too short. Make sure you break the line at the last position possible